server/portal/apps/workspace/api/views.py

```diff
@@ -93,10 +93,17 @@ def get(self, request, *args, **kwargs):
         else:
             METRICS.debug("user:{} is requesting all public apps".format(request.user.username))
             public_only = request.GET.get('publicOnly')
+            name = request.GET.get('name', None)
+            list_kwargs = {}
             if public_only == 'true':
-                data = agave.apps.list(publicOnly='true')
+                list_kwargs['publicOnly'] = 'true'
```

Comment:
Should this be `True` instead of `str` `"true"`?
```suggestion
                list_kwargs['publicOnly'] = True
```

Reply:
unfortunately, no? The original code passed true as a string instead of True.

Reply:
`publicOnly` takes a string, while `privateOnly` takes a bool?